.github/workflows/mypy.yaml

```diff
@@ -0,0 +1,45 @@
+name: Mypy CI
+on:
+  push:
+    branches:
+      - develop
+      - master
```

Comment:
We do not have a master branch anymore for MDAnalysis. You can read the discussion about this here: https://github.com/MDAnalysis/mdanalysis/issues/3542

Reply:
I removed master branch